test/acceptance/service-fabrik-admin.internal-db.lifecycle.spec.js

```diff
@@ -28,6 +28,7 @@ describe('service-fabrik-admin', function () {
 
     before(function () {
       config.mongodb.provision.plan_id = 'bc158c9a-7934-401e-94ab-057082a5073f';
+      mocks.director.getBindingProperty(CONST.FABRIK_INTERNAL_MONGO_DB.BINDING_ID, {}, config.mongodb.deployment_name, 'NOTFOUND');
```

Comment:
'NOTFOUND' --> can we define a constant? or fine ??

Reply:
Thats the name of the error class, no need to define it in constants.